receiver/mysqlreceiver/scraper.go

```diff
@@ -271,6 +271,50 @@ func (m *mySQLScraper) scrape(context.Context) (pmetric.Metrics, error) {
 		}
 	}
 
+	// collect table_io_waits metrics.
+	tableIoWaitsStats, err := m.sqlclient.getTableIoWaitsStats()
+	if err != nil {
+		m.logger.Error("Failed to fetch table io_waits stats", zap.Error(err))
+		return pmetric.Metrics{}, err
```

Comment:
We should not fail out at this point. Instead, add to the partialError and continue collecting what can be collected. If nothing else, we need to make sure we are not dropping any previous partial errors.